daemon/inspect.go

```diff
@@ -59,7 +66,22 @@ func (daemon *Daemon) ContainerInspect(name string) (*types.ContainerJSON, error
 		StartedAt:  container.State.StartedAt,
 		FinishedAt: container.State.FinishedAt,
 	}
-
+	volumes := make(map[string]string)
+	volumesRW := make(map[string]bool)
+	for _, v := range container.volumes {
+		config := container.VolumeConfig[v.Name()]
+		path, err := v.Mount()
+		if err != nil {
+			return nil, err
+		}
+		volumes[config.Destination] = path
+		volumesRW[config.Destination] = config.RW
+		v.Unmount()
```

Comment:
Different drivers can implement it in a different way.  Mount then an Unmount lets the driver know that this volume is in use and when docker is done using the volume.